recipes/wand/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "Wand" %}
+{% set version = "0.4.4" %}
+{% set sha256 = "28e0454c9d16d69c5d5034918d96320d8f9f1377b4fdaf4944eec2f938c74704" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+  run:
```

Comment:
Should `imagemagick` or `magickwand` be a requirement?

http://docs.wand-py.org/en/0.4.4/#requirements

Reply:
I think you are right. I looked at the CI details, and looks like the tests are not running.